hikari/emojis.py

```diff
@@ -199,6 +199,12 @@ def unicode_names(self) -> typing.Sequence[str]:
 
         This returns the name of each codepoint. If only one codepoint exists,
         then this will only have one item in the resulting sequence.
+
+        !!! warning
+            This uses `unicodedata.name` to fetch the names. This list
+            only gets updated between python versions. This means
+            that this might raise `builtins.ValueError` if there is
```

Comment:
I feel like this could be better worded, something like "this method relies on the standard unicodedata module and thus is limited by the unicode version python knows" (also might make sense to call it a newer unicode version rather than a "new version of emojis")